samples/Ralltiir/Ralltiir.Server/Ralltiir.Server.csproj

```diff
@@ -0,0 +1,54 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.AspNetCore.Identity" Version="2.2.0" />
```

Comment:
Please don't specify the versions in this file. Instead, put them in `Packages.props`. Feel free to take a look at the other samples to see how it's done.

Reply:
I honestly didn't know you could do that ^_^